src/static/js/modules/pagination-validation.js

```diff
@@ -0,0 +1,50 @@
+/* ==========================================================================
+   Pagination Form validation
+   Check to verify page input value is number and within min/max range
+   before submitting form.
+   ========================================================================== */
+
+'use strict';
+
+// Import modules
+var $ = require( 'jquery' );
+
+/**
+ * Set up DOM references and event handlers.
+ */
+function init() {
+  $( '.js-validate_pagination' ).each( function() {
+    var $form = $( this );
+    var $placeHolder = $( '<div class=\'cf-notification__pagination-ctr\'></div>' );
+    $form.parents( 'nav.pagination' ).before( $placeHolder );
+    $( $placeHolder ).cf_notifier();
```

Comment:
You can attach `cf_notifier` to the form, no need for a placeholder. Then on submit, trigger the `cf_notifier:notify` event with your message in the options (if it's different from your default message).